README.md

```diff
@@ -1,2 +1,228 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 NYALIK PORTFOLIO <a name="about-project"></a>
+
+
+**nyalik portfolio** is a simple project to demonstrate my competence as a web developer by literally showcasing my development skills.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is entirely built with HTML & CSS
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **An Introductory session with basic highlight**
+- **A portfolio section that shows my previous projects**
+- **A contact session to get in touch with me**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+1. Clone this repository
+   run 
+   **git clone git@github.com:marvin-nyalik/NyalikPortfolio.git**
+
+2. Open the project in a text editor
+3. Serve using a local development server e.g **liveserver** or **apache** 
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **MARVIN OTIENO**
+
+- GitHub: [@githubhandle](https://github.com/marvin-nyalik/)
+- Twitter: [@twitterhandle](https://twitter.com/NyalikMarvin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marvin-otieno-05ba83263/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
```

Comment:
- [ ] For clean code and as it is mentioned in this README template, kindly ensure that you remove all the comments and instructions after you have finished to use it. Make sure to apply this this to the whole file